cmd/lncli/cmd_invoice.go

```diff
@@ -82,8 +154,20 @@ var addInvoiceCommand = cli.Command{
 			Usage: "creates an AMP invoice. If true, preimage " +
 				"should not be set.",
 		},
+		cli.StringFlag{
+			Name: "route_hints",
+			Usage: "Base64-encoded string representing route " +
+				"hints for the invoice. Each route hint " +
+				"should be a JSON object with fields: " +
+				"node_id (string), chan_id (number), " +
+				"fee_base_msat (number, optional), " +
+				"fee_proportional_millionths " +
+				"(number, optional), cltv_expiry_delta " +
+				"(number, optional). Encode your JSON route " +
+				"hints into Base64 before using this flag.",
```

Comment:
doesnt really make sense to me to force the user to first encode to base 64 before calling this? 

why not let them specify the routing hints as a JSON string as you have shown in the test you added?

Reply:
I ended up base64-encoding the input because I was consistently running into issues passing the json string over bash. I feel like the base64 structure provides a bit more integrity to the input. It definitely adds more of a hassle to the end-user but I found it to be the only way to pass these complex json strings without errors popping up.

Reply:
Nvm was able to get it working! Will tweak this to take in the JSON string instead 👍